tutorials/Tutorial16_Document_Classifier_at_Index_Time.py

```diff
@@ -21,70 +23,150 @@ def tutorial16_document_classifier_at_index_time():
     s3_url = "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-qa/datasets/documents/preprocessing_tutorial.zip"
     fetch_archive_from_http(url=s3_url, output_dir=doc_dir)
 
-    # read and preprocess documents
+
+    # ## Read and preprocess documents
+
     # note that you can also use the document classifier before applying the PreProcessor, e.g. before splitting your documents
-    all_docs = convert_files_to_dicts(dir_path="data/preprocessing_tutorial")
+    all_docs = convert_files_to_dicts(dir_path=doc_dir)
     preprocessor_sliding_window = PreProcessor(
         split_overlap=3,
         split_length=10,
         split_respect_sentence_boundary=False
     )
     docs_sliding_window = preprocessor_sliding_window.process(all_docs)
 
-    """ 
-    
-    ## DocumentClassifier
+    # ## Apply DocumentClassifier
+
+    # We can enrich the document metadata at index time using any transformers document classifier model.
+    # Here we use a zero shot model that is supposed to classify our documents in 'music', 'natural language processing' and 'history'.
+    # These classes can later on be accessed at query time.
```

Comment:
As not all of our users might know what zero shot classification is, please motivate a bit how cool it is :) . Something like: "While traditional classification models are trained to predict one of a few "hard-coded" classes and required a dedicated training dataset, zero-shot classification is super flexible and you can easily switch the classes the model should predict on the fly. Just supply them via the `labels` param.

Reply:
motivation mode on: done :-)